lib/components/DataStorageCheck.js

```diff
@@ -0,0 +1,44 @@
+/*
+* == BSD2 LICENSE ==
+* Copyright (c) 2016, Tidepool Project
+*
+* This program is free software; you can redistribute it and/or modify it under
+* the terms of the associated License, which is identical to the BSD 2-Clause
+* License as published by the Open Source Initiative at opensource.org.
+*
+* This program is distributed in the hope that it will be useful, but WITHOUT
+* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+* FOR A PARTICULAR PURPOSE. See the License for more details.
+*
+* You should have received a copy of the License along with this program; if
+* not, you can obtain one from Tidepool Project at tidepool.org.
+* == BSD2 LICENSE ==
+*/
+
+var _ = require('lodash');
```

Comment:
We don't really have any plans for converting old code. The idea has just been to move forward with ES6 and perhaps make a judgment call when making modifications to old code. That is, if the modifications seem large enough, port to ES6. If small, just make changes in ES5.

Here in the uploader we also have to be a bit careful because the CLI tools used for uploading from certain file types need ES5 (due to our continued use of node v0.12, though of course that may change soon as well). So we have to be careful that anything in the require chain for the CLI tools does not use ES6 things that node v0.12 doesn't support. This is actually broken on master, but @gniezen has patched on his Medtronic branch.